components/ProductCard/index.js

```diff
@@ -1,14 +1,63 @@
 import styled from "styled-components";
+import Counter from "../Counter";
 
-export default function ProductCard() {
-  return <StyledProductCard>1234</StyledProductCard>;
+//each product card component
+export default function ProductCard(props) {
+  return (
+    <StyledProductCard>
+      <StyledImg
+        src={props.productcard.image}
+        alt="photo"
+        width="180px"
+        height="180px"
+      />
+      <div>
+        <StyledTitle>{props.productcard.title}</StyledTitle>
+        <StyledDes>{props.productcard.description}</StyledDes>
+        <StyledPrice>{props.productcard.price}</StyledPrice>
+      </div>
+      <div><Counter amount={props.productcard.amount} /></div>
+    </StyledProductCard>
```

Comment:
Might wanna use the fancy ES6 functionality `object desctructuring`

```suggestion
export default function ProductCard({productcard}) {
const {image, title, description, price, amount} = productcard
  return (
    <StyledProductCard>
      <StyledImg
        src={image}
        alt="photo"
        width="180px"
        height="180px"
      />
      <div>
        <StyledTitle>{title}</StyledTitle>
        <StyledDes>{description}</StyledDes>
        <StyledPrice>{price}</StyledPrice>
      </div>
      <div><Counter amount={amount} /></div>
    </StyledProductCard>
```